charts/pgadmin4/values.yaml

```diff
@@ -14,6 +14,11 @@ image:
 ## Deployment annotations
 annotations: {}
 
+## Deployment entrypoint override
```

Comment:
Easier to point towards the Dockerfile here (for the entrypoint command), rather then in the README.md.